src/components/Playbar/index.tsx

```diff
@@ -140,16 +142,47 @@ export function PlaybarUI(props: PlaybarUIProps): ReactElement {
                         className="flex flex-col h-[30px] justify-center flex-shrink-0 cursor-pointer w-[72px] ml-1 group"
                         onClick={onClickCalendar}
                     >
-                        <div className="flex items-end h-[12px] mt-[1px]">
-                            <span className="select-none leading-none text-sm text-blue-600 font-bold uppercase group-hover:text-blue-800 group-active:text-blue-900">
-                                {eventMetaData?.quote?.localTicker || 'Instrument'}
-                            </span>
-                            <span className="select-none truncate leading-none ml-1 mb-[1px] text-xxs uppercase tracking-widest text-gray-400 group-hover:text-gray-600 group-active:text-gray-800">
-                                {eventMetaData?.quote?.exchange?.shortName || 'Exchange'}
-                            </span>
-                        </div>
+                        {isCustom && eventMetaData?.title ? (
+                            <Tooltip
```

Comment:
For custom events that have a title defined in the meta data, display it truncated with ellipsis instead of the default `INSTRUMENT` (when the event isn't tied to an equity), and show the whole title in the tooltip on hover